specification/personal-demographics.yaml

```diff
@@ -1910,14 +1911,29 @@ components:
         example: 'Bearer g1112R_ccQ1Ebbb4gtHBP1aaaNM'
     CorrelationID:
       in: header
-      name: NHSD-Correlation-ID
+      name: X-Correlation-ID
       required: true
       description:
-        A unique identifer should be created when making a request, this will assist with any problems that may occur and should be included when an issue is raised with the helpdesk.
+        An identifer should be created when making a request, this will assist with any problems that may occur and should be included when an issue is raised with the helpdesk.
```

Comment:
Disallow `.` characters to avoid breaking the packing/unpacking... or use spaces to separate or multiple repeated header values if they have an order that is preserved?

Reply:
Added a disclaimer.